eduvpn/crypto.py

```diff
@@ -14,61 +14,37 @@
 
 
 def common_name_from_cert(pem_data):
-    """
-    Extract common name from client certificate.
-
-    args:
-        pem_data (str): PEM encoded certificate
-
-    returns:
-        str: the common name of the client certificate.
-
-    """
-    cert = x509.load_pem_x509_certificate(pem_data, default_backend())
-    return cert.subject.get_attributes_for_oid(NameOID.COMMON_NAME)[0].value
+	#type: (str) -> str
```

Comment:
now the arguments don't have a doc string anymore. You can leave the doc and just remove the types.

Reply:
fixed in all files.